content/docs/engine-dev/tools/tiaf/maintaining-tiaf/general-information.md

```diff
@@ -0,0 +1,114 @@
+
+This method will remove TIAF completely from AR. It should be used as a method of last resort as it requires intrusive edits to the AR Jenkins configuration.
+
+1. Navigate to `scripts\build\Platform\Windows\build_config.json`.
+2. Locate `profile_pipe` and under steps remove `test_impact_analysis_profile_native` and `test_impact_analysis_profile_python`.
+3. Locate `test_cpu_profile` and under `CTEST_OPTIONS` change `(REQUIRES_gpu|REQUIRES_tiaf)` to `(REQUIRES_gpu)`.
+
+# How to I enrol a test into TIAF?
+
+All tests enrolled into the TIAF are run by the appropriate TIAF runtime (`native` for native C++ tests, `python` for Python tests) instead of CTest. Likewise, all tests *not* enrolled into the TIAF are run instead by CTest. In order to enrol a test target into TIAF, simply add the `REQUIRES_tiaf` label when registering the test, like so:
+
+```
+ly_add_googletest(
+    NAME AZ::MyExample.Tests
+    LABELS REQUIRES_tiaf
+)
+```
+
+{{< caution >}}
+It is advised that you place your `LABELS` after the test target name and before any other custom attributes.
+{{< /caution >}}
+
+# How to I enrol a native test for test sharding optimization?
+
+Test sharding is an optional optimization that can greatly boost the speed at which test targets are run (see XXX for more information). As the question implies, the test sharding optimization is only available for native tests. In order to opt a test target into test sharding, simply add either  `TIAF_shard_test` or `TIAF_shard_fixture` label when registering the test, like so:
+
+```
+ly_add_googletest(
+    NAME AZ::MyExample.Tests
+    LABELS REQUIRES_tiaf;TIAF_shard_test
+)
+
+```
+
+When using the `TIAF_shard_test` label, each individual test will be interleaved across the available shards. This delivers the greatest performance boost but increases the brittleness of the sharded test target as resource race conditions for badly behaving tests are more likely to manifest. On the contrary, `TIAF_shard_fixture` interleaves each individual fixture across the available shards, delivering less performance than `TIAF_shard_test` but less brittleness. Always test your targets when enrolling them into test sharding optimization to ensure that they behave well when running as shards.
+
+{{< note >}}
+The test target must be enrolled into native TIAF in order to benefit from test sharding, else the test target will not be run by TIAF.
+{{< /note >}}
+
+# What are the command line options for the TIAF runtimes?
+
+The following table details the command line options common to both the native and Python runtimes:
+
+|Option|Flag|Default|Description|
+|:----|:----|:----|:----|
+|Configuration File|-config=<filename>|<tiaf binay build dir>.<tiaf binary build type>.json|Path to the configuration file for the TIAF runtime.|
+|Test Impact Data File|-testimpactdatafile=<filename>|None|Optional path to a test impact data file that will used instead of that specified in the config file.|
```

Comment:
update (repeated): that will used -> that will be used

Reply:
Fixed 👍